flask-homework/my_app/__init__.py

```diff
@@ -0,0 +1,17 @@
+from flask import Flask
+from logging.config import dictConfig
+from .config import AppConfig
+
+app = Flask(__name__)
+
+dictConfig({
+    'version': 1,
+    'formatters': {'default': {
+        'format': '[%(asctime)s] %(levelname)s in %(module)s: %(message)s'
+    }},
+})
+
+app.config.from_object(AppConfig)
+
+from .views import *
+from .class_base_views import *
```

Comment:
There is no such file `class_base_views`

Reply:
Yeah, write! That was me trying to implement class-based viewes. And I forgot to delete this line after unsuccessful attempts. Will try again using your hints within the next homework